src/ensembl/src/shared/helpers/numberFormater.test.ts

```diff
@@ -0,0 +1,31 @@
+import { getCommaSeparatedNumber } from './numberFormatter';
+
+describe('getCommaSeparatedNumber', () => {
+  it('returns 1234 for the input 1234', () => {
```

Comment:
1) incorrect description
2) too specific description (what's so special about sequence 1234?); it would be better if the description said something to the effect of "separates thousand group in a single-thousand number", "separates thousand group in a ten-thousand number", etc. The number can then be generated randomly with faker (`faker.random.number({min:1000, max:9999})`.

Reply:
1. corrected it

2. although it is too specific, I feel it is more easier to understand when we say "it converts 1000 to 1,000" than saying "separates thousand group in a single-thousand number".

Also, if I generate a random number with faker, what will be the value for the expected output?

Reply:
>  if I generate a random number with faker, what will be the value for the expected output

If you generate a random number within a 1000-9999 range, you would expect the resultant string, when split at the comma, to have 1 character in the first string and 3 characters in the last string, wouldn't you? 😉 

(you could even concatenate the result of the split and compare it with the original number, and make sure they are the same 😄 )

Reply:
The first method will work but are you sure we should do this? It feels like we might need to come up with another test to test if this test works fine 😄 

The second will give false positive if the formatted result is 12,34.

Reply:
But the first and the second checks will be done together.

`1234` will be formatted as `1,234`, which will split into a single-digit group (`'1'`) and as three-digit group `'234'`, which, when joined together, will result in `'1234'`, which will be equal to stringified `1234`.

Isn't it the operational definition of what we are doing to these numbers?

Reply:
The tests are now updated.